src/app/core/entity-components/entity-utils/dynamic-form-components/edit-single-entity/edit-single-entity.component.ts

```diff
@@ -23,7 +28,10 @@ export class EditSingleEntityComponent extends EditComponent<string> {
 
   @ViewChild("inputElement") input: ElementRef;
 
-  constructor(private dynamicEntityService: DynamicEntityService) {
+  constructor(
+    private dynamicEntityService: DynamicEntityService,
+    private changeDetection: ChangeDetectorRef
```

Comment:
do we really, really need to handle change detection manually here?

Reply:
Unfortunately the asynchronous function makes this necessary